modules.d/91crypt-gpg/module-setup.sh

```diff
@@ -5,6 +5,13 @@
 check() {
     require_binaries gpg || return 1
 
+    if sc_requested; then
+        if ! sc_supported; then
+            dwarning "crypt-gpg: GnuPG >= 2.1 with scdaemon and libusb required for ccid smartcard support"
+            return 1
+        fi
```

Comment:
you might want to "return 0" here

"return 255" means, that the module is only included, if it is specified manually or required by other modules.

But sc_reqeusted() checks for a user configured file, so it would be save to include the module regardless.

What do you think?